tests/expressions/tests.py

```diff
@@ -1828,3 +1828,14 @@ def test_reversed_and(self):
     def test_reversed_or(self):
         with self.assertRaisesMessage(NotImplementedError, self.bitwise_msg):
             object() | Combinable()
+
+
+class ExpressionWrapperGroupByTests(SimpleTestCase):
+
+    def test_empty_group_by(self):
+        expr = ExpressionWrapper(Value(3), output_field=IntegerField())
+        self.assertEqual(expr.get_group_by_cols(alias=None), [])
+
+    def test_non_empty_group_by(self):
+        expr = ExpressionWrapper(Value(3) + Value(5), output_field=IntegerField())
+        self.assertEqual(expr.get_group_by_cols(alias=None), [expr.expression])
```

Comment:
We should test with a different expression since this might be fixed in the future

```suggestion
        expr = ExpressionWrapper(Lower('field'), output_field=IntegerField())
        self.assertEqual(expr.get_group_by_cols(alias=None), [expr.expression])
```